compiler/optimizer/Inliner.cpp

```diff
@@ -4811,6 +4845,11 @@ bool TR_InlinerBase::inlineCallTarget2(TR_CallStack * callStack, TR_CallTarget *
       }
 
 
+#ifdef J9_PROJECT_SPECIFIC
+   // We have the trees now, we can check if each argument is invariant and clear the prex arg for the ones that are not
+   calltarget->_prexArgInfo->clearArgInfoForNonInvariantArguments(calltarget->_calleeSymbol, tracer());
```

Comment:
Could this be moved into a Util function to avoid the `ifdef`?

Reply:
Done.